src/Consts/excercises.js

```diff
@@ -1,3 +1,15 @@
+<<<<<<< HEAD
```

Comment:
something strange is here, proper content starts at 
const = { 
    class0: { .....

Reply:
Merging issue, to be fixed upon next merge with development.